### PR TITLE
Laughter demons provide explosion protection

### LINKED ISSUE #18979: laughter demon ex_act
1. Be laughter demon (honk)
2. Eat a ton of people
3. Die to explosion (My case was immovable rod, it would do this for sure)
4. Watch as limbs fly out of you and then after that lag you die
5. Everyone is revived... without their limbs and most likely dead from an immovable rod calling ex_act on them
6. Deadchat erupts in salt, pepper, laughter, and all kinds of crazy

EDIT: 7. Admins realize that LIMBS HAVE ALSO TELEPORTED ALL OVER THE PLACE

### PR DESCRIPTION
**Pull request by coiax**
Fixes #18979. You're supposed to be in magical hugspace, not having your
limbs torn off.

### CODE REVIEW

File: code/game/gamemodes/miniantags/slaughter/slaughter.dm [hunk 1]
```diff
@@ -150,22 +150,17 @@
...
+	switch(severity)
+		if(1)
+			death()
+		if(2)
+			adjustBruteLoss(60)
+		if(3)
+			adjustBruteLoss(30)
 
 /mob/living/simple_animal/slaughter/laughter/proc/release_friends()
 	if(!consumed_mobs)
```
Comment (Supermichael777) [inline code/game/gamemodes/miniantags/slaughter/slaughter.dm]: Unrelated to change but is this necessary?  For loops are one of the few bits of code i still don't understand the logic of(outside of base function). wouldn't a loop with an empty list just instantly finish?

Comment (RemieRichards) [inline code/game/gamemodes/miniantags/slaughter/slaughter.dm]: @Supermichael777 you seem to understand it more than you think, yes this line (and the return below it) are unnecessary, the for loop won't run on an empty list, and it certainly won't error or anything bad.

Comment (coiax) [inline code/game/gamemodes/miniantags/slaughter/slaughter.dm]: This is checking if the list is null though, not if its empty

**Comment by Zilenan91** [comment]
Why though that was really funny

**Comment by kevinz000** [comment]
you're welcome for the token

**Comment by jughu** [comment]
Revert

**Pull request merged**